host-bin/enter-chroot

```diff
@@ -33,6 +34,7 @@ Options:
     -n NAME     Name of the chroot to enter. Default: first one found in CHROOTS
     -t TARGET   Only enter the chroot if it contains the specified TARGET.
     -u USERNAME Username (or UID) to log into. Default: 1000 (the primary user)
+    -X XMETHOD  For this session, override the the auto-detected XMETHOD
```

Comment:
@DennisLfromGA needs to make the changes in the source, but you're welcome to keep trying in the comment box.